packages/server/graphql/mutations/jiraCreateIssue.ts

```diff
@@ -0,0 +1,123 @@
+import {GraphQLID, GraphQLNonNull, GraphQLString} from 'graphql'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import {IJiraCreateIssueOnMutationArguments} from 'parabol-client/types/graphql'
+import AtlassianServerManager from '../../utils/AtlassianServerManager'
+import {getUserId, isTeamMember} from '../../utils/authorization'
+import publish from '../../utils/publish'
+import segmentIo from '../../utils/segmentIo'
+import standardError from '../../utils/standardError'
+import {GQLContext} from '../graphql'
+import JiraCreateIssuePayload from '../types/JiraCreateIssuePayload'
+
+export default {
+  name: 'JiraCreateIssue',
+  type: JiraCreateIssuePayload,
+  args: {
+    cloudId: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'The atlassian cloudId for the site'
+    },
+    meetingId: {
+      type: GraphQLID,
+      description:
+        'The id of the meeting where the Jira issue is being created. Null if it is not being created in a meeting.'
+    },
+    projectKey: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'The atlassian key of the project to put the issue in'
+    },
+    summary: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'The text content of the Jira issue'
+    },
+    teamId: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'The id of the team that is creating the issue'
+    }
+  },
+  resolve: async (
+    _source: object,
+    {cloudId, meetingId, projectKey, summary, teamId}: IJiraCreateIssueOnMutationArguments,
+    {authToken, dataLoader, socketId: mutatorId}: GQLContext
+  ) => {
+    const operationId = dataLoader.share()
+    const subOptions = {mutatorId, operationId}
+    const viewerId = getUserId(authToken)
+
+    // AUTH
+    if (!isTeamMember(authToken, teamId)) {
+      return standardError(new Error('Team not found'), {userId: viewerId})
+    }
+
+    // VALIDATION
```

Comment:
to validate:
- does a meetingMember for the viewer with the provided meetingId exist?
- if meetingId is provided, is the meeting owned by the provided teamId?